api-reference/echo-operations/README.md

```diff
@@ -91,11 +91,11 @@ transaction. This off-chain synchronization can be avoided using proposals.
 
```

Comment:
Add a brief description of the operations:
1. contract_whitelist_operation
2. to all "Sidechain" operations

Reply:
Fixed